modules/reference/pages/rpk/rpk-registry/rpk-registry-schema-get.adoc

```diff
@@ -29,6 +31,8 @@ rpk registry schema get [SUBJECT] [flags]
 
 |--id |int | Schema ID to look up usage; subject optional.
 
+|--print-schema |- |When this flag is used, print the schema in JSON format.

```

Comment:
this is different from the generated content: https://github.com/redpanda-data/docs/pull/285/files#diff-1596ce07c033aa828075b3650363061083af0d6171b5ce2b44375f2a56add51b

'When this flag is used' is superfluous. We could say 'Print the schema (to the console?) in JSON format'

Reply:
Yeah I edited a bit from the generated content. Looking to maybe port this change to code after the reviews.

On code, i missed in on my review
https://github.com/redpanda-data/redpanda/pull/16109/files#diff-183f291208601de2ac00c3ffc73adab1c72a3efb82176d8523fcd344e1b81cc4R140

Reply:
```suggestion
|--print-schema |- |Prints the schema in JSON format.

```

Reply:
"to console" is prob superfluous too since you're already on the CLI